src/app/core/components/header/header.component.scss

```diff
@@ -0,0 +1,234 @@
+            .icon{
+                width: 30px;
+                height: 24px;
+                margin-right: 7px;
+                vertical-align: middle;
+            }
+            span{
+                font-family: $font-primary;
+                font-weight: $font-weight-semibold;
+                color: #2D3134;
+            }
+        }
+        .menu-toggle {
+            display: none;
+            background: none;
+            border: none;
+            font-size: 24px;
+            cursor: pointer;
+            @media (max-width: 1040px) {
+                display: block;
+            }
+        }
+        @media (max-width: 1040px) {
+            padding: 20px;
+            justify-content: space-between;
+        }
+        .menu{
+            .navbar{
+                margin-left: 60px;
+                a{
+                    position: relative;
+                    font-family: $font-primary;
+                    font-size: 14px;
+                    color: #2E476B;
+                    font-weight: $font-weight-regular;
+                    margin: 0 20px;
+                    text-decoration: none;
+                    cursor: pointer;
+                    &:hover, &:active{
+                        color: #36B0DC;
+                    }
+                    &::after{
+                        content: '';
+                        position: absolute;
+                        left: 0;
+                        top: 20px;
+                        width: 0;
+                        height: 2px;
+                        background: #36B0DC;
+                        opacity: 0;
+                        transition: width 0.3s ease;
+                    }
+                    &:hover::after{
+                        opacity: 1;
+                        width: 30%;
+                    }
+                    &.regions-nav {
+                        @media (max-width: 1370px) {
+                            display: none;
+                        }
+                    }
+                }
+                @media (max-width: 1040px) {
+                    display: none;
+                    flex-direction: column;
+                    &.active {
+                        display: flex;
+                    }
+                    a {
+                        margin: 10px 0;
+                        right: 20px;
+                        top: 20px;
+                    }
+                }
+            }
+            @media (max-width: 1040px) {
+                position: absolute;
+                right: 0;
+                top: 70px;
+                background: #fff;
+                width: 200px;
+                height: 210px;
+                opacity: 0;
+                &.active {
+                    opacity: 1;
+                }
+            }
+        }
+        
+    }
+    .profile-container {
+        position: relative;
+        .image {
+            width: 50px;
+            height: 50px;
+            margin-left: 250px;
```

Comment:
this margin is unnecessary - let's try avoiding this and cleaning up the entire stylesheet where possible